src/lib/plugin_apis/fs.api

```diff
@@ -334,6 +334,86 @@ GType bd_fs_ntfs_info_get_type () {
     return type;
 }
 
+#define BD_FS_TYPE_F2FS_INFO (bd_fs_f2fs_info_get_type ())
+GType bd_fs_f2fs_info_get_type();
+
+/**
+ * BDFSF2FSInfo:
+ * @label: label of the filesystem
+ * @uuid: uuid of the filesystem
+ * @cluster_size: cluster size used by the filesystem
+ * @cluster_count: number of clusters in the filesystem
+ * @free_cluster_count: number of free clusters in the filesystem
```

Comment:
`free_cluster_count` described here, but `features` is part of the struct instead.

Reply:
Fixed., the `cluster` attributes were also wrong.